operate/cli.py

```diff
@@ -940,6 +947,101 @@ def _daemon(
     server.run()
 
 
+@_operate.command(name="quickstart")
+def quickstart(
+    config: Annotated[str, params.String(help="Quickstart config file path")],
+) -> None:
+    """Quickstart."""
+    operate = OperateApp()
+    operate.setup()
+    run_service(operate=operate, config_path=config)
+
+
```

Comment:
For these group of commands, I would suggest calling the functions as:

def quickstart
def qs_stop
def qs_start
def qs_terminate
etc ...

and the dommands without the 'quick' prefix: start stop, etc... It looks a bit strange for the user calling a command "quickstop"